sdk/iot/iot-modelsrepository/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2021-05-24)
```

Comment:
suppose we need to bump this now?

Reply:
to a new version?

Reply:
The date is yesterday, but the package hasn't been published yet as far as I can tell, so we should have the date match when it actually is going to be published.